src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer1 = new Bulldozer("bulldozer 1", "green", "non-rotating");
+        Bulldozer bulldozer2 = new Bulldozer("bulldozer 2", "black", "rotating");
+        List<Bulldozer> bulldozers = new ArrayList<>();
+        bulldozers.add(bulldozer1);
+        bulldozers.add(bulldozer2);
```

Comment:
you can create the list using
 `List<Bulldozer> bulldozers = List.of( 
new Bulldozer("bulldozer 1", "green", "non-rotating"),
new Bulldozer("bulldozer 2", "black", "rotating"));`